static/simple-portfolio-template/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "simple-portfolio-template",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "@tailwindcss/forms": "^0.5.2",
+    "prettier-plugin-tailwindcss": "^0.1.11",
```

Comment:
Subho has added prettier-plugin at the root of the project, so we can remove it from here.

Reply:
Correct. Will make this change.